cmd/juju/charmhub/infowriter_test.go

```diff
@@ -21,13 +21,47 @@ var _ = gc.Suite(&printInfoSuite{})
 func (s *printInfoSuite) TestCharmPrintInfo(c *gc.C) {
 	ir := getCharmInfoResponse()
 	ctx := commandContextForTest(c)
-	iw := makeInfoWriter(ctx.Stdout, ctx.Warningf, false, "never", &ir)
+	iw := makeInfoWriter(ctx.Stdout, ctx.Warningf, false, "never", baseModeBoth, &ir)
 	err := iw.Print()
 	c.Assert(err, jc.ErrorIsNil)
 
 	obtained := ctx.Stdout.(*bytes.Buffer).String()
 	expected := `name: wordpress
-publisher: Wordress Charmers
+publisher: WordPress Charmers
+summary: WordPress is a full featured web blogging tool, this charm deploys it.
+description: |-
+  This will install and setup WordPress optimized to run in the cloud.
+  By default it will place Ngnix and php-fpm configured to scale horizontally with
+  Nginx's reverse proxy.
+charm-id: charmCHARMcharmCHARMcharmCHARM01
+supports: bionic, xenial
+tags: app, seven
+subordinate: true
+relations:
+  provides:
+    one: two
+    three: four
+  requires:
+    five: six
+channels: |
+  latest/stable:     1.0.3  2019-12-16  (16)  12MB  amd64  jammy, focal
```

Comment:
For status and show-machine / show-application, we're now transitioning away from series to display base in tabular output, ie "ubuntu:22.04" etc.
I wonder if we want to do the same here. Charmhub shows the version numbers in its UX. We should be consistent with that I think. We could use a similar UX, eg "ubuntu: 18.04 20.04 22.04"